ext/openssl/tests/openssl_password.phpt

```diff
@@ -0,0 +1,42 @@
+--TEST--
+Basic features of password_hash
+--EXTENSIONS--
+openssl
+--SKIPIF--
+<?php
+if (!function_exists('openssl_password_hash')) {
+  echo "skip - No openssl_password_hash";
```

Comment:
all other tests (or most of them) use 4 spaces indent so might be nice to keep it consistent